Do not add LDFLAGS to pkg-config flags, let user use its own

bicatali@gentoo.org
--- libstarpu.pc.in.orig	2012-05-29 23:16:20.000000000 +0100
+++ libstarpu.pc.in	2012-05-30 01:09:41.000000000 +0100
@@ -24,6 +24,6 @@
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}/starpu/@STARPU_EFFECTIVE_VERSION@ @STARPU_CUDA_CPPFLAGS@ -DSTARPU_USE_DEPRECATED_API
 Libs: -L${libdir} -lstarpu-@STARPU_EFFECTIVE_VERSION@ @STARPU_CUDA_LDFLAGS@ @STARPU_OPENCL_LDFLAGS@
-Libs.private: @LDFLAGS@ @LIBS@
+Libs.private: @LIBS@
 Requires: @HWLOC_REQUIRES@
 Requires.private: @GORDON_REQUIRES@
--- mpi/libstarpumpi.pc.in.orig	2012-05-29 23:16:20.000000000 +0100
+++ mpi/libstarpumpi.pc.in	2012-05-30 01:09:53.000000000 +0100
@@ -24,6 +24,6 @@
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}/starpu/@STARPU_EFFECTIVE_VERSION@ -DSTARPU_USE_DEPRECATED_API
 Libs: -L${libdir} -lstarpumpi-@STARPU_EFFECTIVE_VERSION@
-Libs.private: @LDFLAGS@ @LIBS@
+Libs.private: @LIBS@
 Requires: libstarpu
 Requires.private:
--- mpi/starpumpi-1.0.pc.in.orig	2012-05-29 23:16:20.000000000 +0100
+++ mpi/starpumpi-1.0.pc.in	2012-05-30 01:10:11.000000000 +0100
@@ -24,6 +24,6 @@
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}/starpu/@STARPU_EFFECTIVE_VERSION@
 Libs: -L${libdir} -lstarpumpi-@STARPU_EFFECTIVE_VERSION@
-Libs.private: @LDFLAGS@ @LIBS@
+Libs.private: @LIBS@
 Requires: starpu-1.0
 Requires.private:
--- starpufft/starpufft-1.0.pc.in.orig	2012-05-29 23:16:20.000000000 +0100
+++ starpufft/starpufft-1.0.pc.in	2012-05-30 01:10:25.000000000 +0100
@@ -24,4 +24,4 @@
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}/starpu/@STARPU_EFFECTIVE_VERSION@ @STARPU_CUDA_CPPFLAGS@
 Libs: -L${libdir} -lstarpufft-@STARPU_EFFECTIVE_VERSION@ 
-Libs.private: @LDFLAGS@ @LIBS@ @STARPU_CUFFT_LDFLAGS@ @FFTW_LIBS@ @FFTWF_LIBS@
+Libs.private: @LIBS@ @STARPU_CUFFT_LDFLAGS@ @FFTW_LIBS@ @FFTWF_LIBS@
--- starpufft/libstarpufft.pc.in.orig	2012-05-29 23:16:20.000000000 +0100
+++ starpufft/libstarpufft.pc.in	2012-05-30 01:10:43.000000000 +0100
@@ -24,4 +24,4 @@
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}/starpu/@STARPU_EFFECTIVE_VERSION@ @STARPU_CUDA_CPPFLAGS@ -DSTARPU_USE_DEPRECATED_API
 Libs: -L${libdir} -lstarpufft-@STARPU_EFFECTIVE_VERSION@ 
-Libs.private: @LDFLAGS@ @LIBS@ @STARPU_CUFFT_LDFLAGS@ @FFTW_LIBS@ @FFTWF_LIBS@
+Libs.private: @LIBS@ @STARPU_CUFFT_LDFLAGS@ @FFTW_LIBS@ @FFTWF_LIBS@
--- starpu-1.0.pc.in.orig	2012-05-29 23:16:20.000000000 +0100
+++ starpu-1.0.pc.in	2012-05-30 01:11:06.000000000 +0100
@@ -30,6 +30,6 @@
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}/starpu/@STARPU_EFFECTIVE_VERSION@ @STARPU_CUDA_CPPFLAGS@
 Libs: -L${libdir} -lstarpu-@STARPU_EFFECTIVE_VERSION@ @STARPU_CUDA_LDFLAGS@ @STARPU_OPENCL_LDFLAGS@
-Libs.private: @LDFLAGS@ @LIBS@
+Libs.private: @LIBS@
 Requires: @HWLOC_REQUIRES@
 Requires.private: @GORDON_REQUIRES@
--- socl/socl-1.0.pc.in.orig	2012-05-29 23:16:20.000000000 +0100
+++ socl/socl-1.0.pc.in	2012-05-30 01:11:19.000000000 +0100
@@ -24,6 +24,6 @@
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}/starpu/@STARPU_EFFECTIVE_VERSION@/socl
 Libs: -L${libdir} -lsocl-@STARPU_EFFECTIVE_VERSION@
-Libs.private: @LDFLAGS@ @LIBS@
+Libs.private: @LIBS@
 Requires: starpu-1.0
 Requires.private:
